color/src/colorspace.rs

```diff
@@ -58,6 +58,21 @@ pub trait Colorspace: Clone + Copy + 'static {
         let scaled = LinearSrgb::scale_chroma(rgb, scale);
         Self::from_linear_srgb(scaled)
     }
+
+    /// Clip the color's components to the range allowed by the colorspace.
```

Comment:
I really don't like "allowed by the color space," as my intent is for color spaces to allow all colors. I think I'd phrase this, "to fit within the natural gamut of the color space."

Reply:
Agreed, especially seeing the documentation on `ColorSpace` that it "does not explicitly define a gamut". I changed the language